tests/test_readers.py

```diff
@@ -14,6 +14,15 @@ def test_load_feed():
     assert feed.stop_times.dtypes["arrival_time"] == np.float64
 
 
+def test_load_feed_with_view():
+    full_feed = ptg.load_feed(fixture("trimet-vermont-2018-02-06"))
+    assert full_feed.stops.shape[0] == 102
+
+    view = {"stops.txt": {"stop_id": full_feed.stops.stop_id[0]}}
+    feed = ptg.load_feed(fixture("trimet-vermont-2018-02-06"), view=view)
+    assert feed.stops.stop_id.shape[0] == 72
```

Comment:
Only one stop is present without the fix.

Reply:
I don't think I fully understand, because if the `view` specifies a single `stop_id`, wouldn't it be expected that only that stop is present in the output?

Reply:
Hi Stuart! This is a bit confusing. The reason is because partridge should always preserve the referential integrity of trips. If we consider trips the "atomic unit" of GTFS, then filtering by `stop_id` means "show me all the trips that use this stop". To keep trips whole, the resulting feed will have all stops, stop_times, etc. associated with those trips.

Reply:
Oh, that makes sense. Thanks Danny! :)